lib/optioner.js

```diff
@@ -26,7 +26,9 @@ module.exports = function( callmodule, defaults ) {
   var argv = minimist(process.argv.slice(2))
 
   var sourcemap = {
-    argv:         {},
+    argv:         {
```

Comment:
this is pretty smelly as a fix - what broke and why does it fix it?
it feels like the underlying data structure is degrading in some way